nextflow_schema.json

```
@@ -87,150 +76,193 @@
+          "description": "Run IDs",
+          "help_text": "A comma-separated list of run IDs to run. Use \"All\" to run all samples in the run_metafile."
         },
         "project": {
           "type": "string",
-          "description": "Run samples only from a given project ID"
+          "description": "Project ID",
+          "help_text": "Run samples only from a given project ID"
         },
         "repeat_mapping": {
           "type": "boolean",
-          "description": "If alevin or salmon mapping has already been performed and output files exist, mapping is skipped by default."
+          "description": "Repeat mapping",
+          "help_text": "If alevin or salmon mapping has already been performed and output files exist, mapping is skipped by default."
         },
         "repeat_genetic_demux": {
           "type": "boolean",
-          "description": "If genetic demultiplexing has been performed and output files exist, genetic demux is skipped by default."
+          "description": "Repeat genetic demultiplexing",
+          "help_text": "If genetic demultiplexing has already been performed and output files exist, genetic demux is skipped by default."
         },
         "skip_genetic_demux": {
           "type": "boolean",
-          "description": "Skip genetic demultiplexing steps, even if bulk data is present."
+          "description": "Skip genetic demultiplexing",
+          "help_text": "Skip genetic demultiplexing steps, even if bulk data is present."
         },
         "perform_celltyping": {
           "type": "boolean",
-          "description": "Specify whether or not to incorporate cell type annotations."
+          "description": "Perform cell typing",
+          "help_text": "Cell typing is skipped by default; use this option to incorporate cell type annotations."
         },
         "repeat_celltyping": {
           "type": "boolean",
-          "description": "If cell type annotations already exist, cell type classification with `SingleR` and `CellAssign` is skipped by default."
+          "description": "Repeat cell typing",
+          "help_text": "If cell type annotations have already been performed and output files exist, cell typing is skipped by default."
         },
         "publish_fry_outs": {
           "type": "boolean",
-          "description": "alevin-fry outputs are not published by default. Use this option to publish them to the `checkpoints` folder."
+          "description": "Publish alevin-fry outputs",
+          "help_text": "alevin-fry outputs are not published by default. Use this option to publish them to the `checkpoints` folder."
         },
         "merge_run_ids": {
           "type": "string",
           "default": "All",
-          "description": "Which runs to merge when using the merge.nf workflow.",
+          "description": "Run IDs to merge",
+          "help_text": "A comma-separated list of run IDs to merge. Use \"All\" to merge all runs in the run_metafile.",
```

```suggestion
          "help_text": "A comma-separated list of run IDs to merge. Use \"All\" to merge all runs in the run_metafile. Only used in the merge.nf workflow",
```
I think we want to note that this is only needed for merge.nf, otherwise it's ignored.